cmd/liqoctl/cmd/purge.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019-2022 The Liqo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"time"
+
+	"github.com/spf13/cobra"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
+
+	"github.com/liqotech/liqo/pkg/liqoctl/purge"
+)
+
+const (
+	// liqoctlPurgeShortHelp contains the short help message for purge Liqoctl command.
+	liqoctlPurgeShortHelp = "Purge the Liqo peering between two clusters"
+	// liqoctlPurgeLongHelp contains the long help message for purge Liqoctl command.
+	liqoctlPurgeLongHelp = `Purge the Liqo peering between two clusters.`
+
+	// liqoctlPurgeKubeconfig1Help contains the help message for kubeconfig1 flag.
+	liqoctlPurgeKubeconfig1Help = "The kubeconfig file for the first cluster"
+	// liqoctlPurgeKubeconfig2Help contains the help message for kubeconfig2 flag.
+	liqoctlPurgeKubeconfig2Help = "The kubeconfig file for the second cluster (required when remote-cluster-id is not provided)"
```

Comment:
Should be probably clarified that specifying the kubeconfig or the cluster ID is not equivalent from the unpeering point of view.